packages/webdriverio/src/commands/element/click.js

```diff
@@ -21,24 +21,22 @@
     <div id="someText">I was not clicked</div>
     :click.js
     it('should demonstrate the click command', function () {
-        var myButton = $('#myButton')
+        let myButton = browser.$('#myButton')
```

Comment:
How about doing `const` instead of `let`?

Reply:
makes sense.